NAMESPACE

```diff
@@ -53,6 +53,9 @@ export(getTwitter)
 export(getTwitterToken)
 export(get_sentiment)
 export(get_stopwords)
+export(get_url_param)
+export(get_url_subdomain)
+export(get_url_suffix)
```

Comment:
And there is another concept tld (top level domain), which is different from suffix. Having said that, we need to add get_tld for the tld_extract function from urltools. Also, to be consistent we should also add our wrapper functions for the followings.

- domain - get_url_domain
- fragment - get_url_fragment
- parameters - get_url_parameters
- path - get_url_path
- port - get_url_port
- scheme - get_url_scheme

- get_url_param
- get_url_subdomain
- get_url_suffix
- get_url_tld